css/main.css

```diff
@@ -0,0 +1,35 @@
+body {
+  font-family: Helvetica;
+  text-align: center;
+  color: #ff6347;
+}
+form {
+  margin: 0 auto;
+  display: block;
+}
+input {
+  text-align: center;
+  padding: 1em;
+}
+input[type="submit"] {
+  border: none;
+  padding: 1em;
+  background-color: #ff6347;
+}
+p {
+  display: inline-block;
+  padding: 0.5em;
+}
+#output {
+  margin: 0 auto;
+  width: 80%;
+}
+.Fizz, .butt {
```

Comment:
Rather than bind these to the content being generated, it might be more maintainable to make these a generic classname. If you were to update your code to allow a user to change the values of `Fizz`/`butt` and `Buzz`/`hole`, then it becomes very difficult to anticipate what they'll enter.